api/src/test/java/org/openmrs/ObsTest.java

```diff
@@ -13,14 +13,7 @@
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsNot.not;
 import static org.hamcrest.core.IsNull.nullValue;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThrows;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
it does not convert assertThrows automatically .

Reply:
i am suspecting the new live templates.

Reply:
did you even use the live template in here?
or do you mean the intellij junit 4 to 5 migration tool?

What can help a bit is to exclude a few packages from import suggestions by IntelliJ. So when you let IntelliJ import a method for you it will automatically choose the one from the jupiter package

You need to exclude these
* `groovy.util.GroovyTestCase.*`
* `junit.framework.*`
* `org.junit.Assert.*`
see
<img width="975" alt="Screen Shot 2020-07-12 at 06 44 47" src="https://user-images.githubusercontent.com/4661144/87239130-c3fc5600-c40b-11ea-9ac6-925da96da785.png">

and dont forget to set the `*` imports to a high number like `999`

<img width="976" alt="Screen Shot 2020-07-12 at 06 46 14" src="https://user-images.githubusercontent.com/4661144/87239138-da0a1680-c40b-11ea-9a22-166f022f25ba.png">

when I now migrate using IntelliJ. I can then do a search & replace of `Assertions.` with ``. This of course leads to compile errors which I can then fix with letting IntelliJ fix the imports (one by one; not sure if there is a fix all). WIth the excluded packages setting at the top it chooses the correct static imports :)